logbook.json

```diff
@@ -27,5 +27,9 @@
   {
     "log": "What do you buyin",
     "by": "arghanovanr"
+  },
+  {
+    "log": "Yesterday, my pull request can't be merge cause conflict file it think. So, i made new one.",
```

Comment:
i think this part might need some changes.

Reply:
Okay, i'll change it right away